README.md

```diff
@@ -105,10 +106,18 @@ be built and run.
 > It uses exactly the same build procedure as this package. Be sure
   to `sudo make install` at the end.
 
-###### libuuid
-> Library for generating UUID's
-> Used by various internal subsystems.
-> `sudo apt-get install uuid-dev`
+###### cogserver
+> OpenCog CogServer Network Server.
+> http://github.com/opencog/cogserver
+> It uses exactly the same build procedure as this package. Be sure
+  to `sudo make install` at the end.
+
+###### attention
```

Comment:
It is more related to previous work, but it this comment make me check and I see that `CMakeLists.txt` doesn't check that `attention` and `spacetime` server are installed despite here they are described as dependencies. Is it correct?